desktop-widgets/downloadfromdivecomputer.ui

```diff
@@ -155,33 +155,104 @@
          </item>
          <item row="6" column="0">
           <layout class="QGridLayout" name="DCGrid">
+           <property name="verticalSpacing">
```

Comment:
Visually this does not look too great to be honest:

![Screenshot from 2024-04-21 10-53-23](https://github.com/subsurface/subsurface/assets/4742747/7ed794cb-7d0c-4f6d-a9e6-082b8dcf28b8)

Two ideas for how this can be improved:
- use the 'delete' icon that is already used in the application;
- combine the dive computer name and the delete button in the same visual element.

Quick example for how this can be done (spacing not quite right yet):

![image](https://github.com/subsurface/subsurface/assets/4742747/f24d3042-e0b1-461a-8671-32269e9af303)


```suggestion
               <item row="0" column="0">
+            <widget class="QFrame" name="frameConfigure">
+             <property name="frameShape">
+              <enum>QFrame::StyledPanel</enum>
+             </property>
+             <property name="frameShadow">
+              <enum>QFrame::Raised</enum>
+             </property>
              <layout class="QGridLayout" name="DC1Grid">
               <property name="horizontalSpacing">
                <number>0</number>
@@ -168,6 +175,9 @@
                 <property name="autoDefault">
                  <bool>false</bool>
                 </property>
+                <property name="flat">
+                 <bool>true</bool>
+                </property>
                </widget>
               </item>
               <item row="0" column="1">
@@ -175,12 +185,17 @@
                 <property name="autoDefault">
                  <bool>false</bool>
                 </property>
-               <property name="text">
-                <string>X</string>
+                <property name="flat">
+                 <bool>true</bool>
+                </property>
+                <property name="icon">
+                 <iconset>
+                  <normaloff>:list-remove-icon</normaloff>:list-remove-icon</iconset>
                 </property>
                </widget>
               </item>
              </layout>
+            </widget>
            </item>
        
```

Reply:
Apologies about the email - hope that doesn't create any problems for you.

If this is accepted, I will remove the Delete All from the preference page.

I agree that your layout looks better.   I don't really like the sizing, but am not sure how to fix that.